batch/batch/client.py

```diff
@@ -111,11 +111,23 @@ def delete(self):
 
 
 class BatchClient:
-    def __init__(self, url=None, timeout=None, cookies=None, headers=None):
+    def __init__(self, url=None, timeout=None, token_file=None, token=None, headers=None):
+        assert token_file is None or token is None
```

Comment:
Assert is not none

Reply:
we're verifying that only one of the two are set, I'll add an error message